packages/editor-ui/src/components/Node.vue

```diff
@@ -228,6 +326,7 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 		return {
 			isTouchActive: false,
 			nodeSubtitle: '',
+			showWebhookNodeTooltip: false,
```

Comment:
we should rename this to `showTriggerNodeTooltip` instead of webhook

Reply:
Renamed.